src/platform/Ameba/DiagnosticDataProviderImpl.cpp

```diff
@@ -273,7 +273,7 @@ CHIP_ERROR DiagnosticDataProviderImpl::GetWiFiBssId(ByteSpan & BssId)
 CHIP_ERROR DiagnosticDataProviderImpl::GetWiFiVersion(uint8_t & wifiVersion)
```

Comment:
As a followup: shouldn't this outparam just be of type `WiFiNetworkDiagnostics::WiFiVersionType`?

Reply:
I opened #24948.